tests/core/gadget-config-defaults-to-snaps/task.yaml

```diff
@@ -84,6 +91,9 @@ prepare: |
     # wait for first boot to be done
     wait_for_first_boot_change
 
+    # install snap with default-configure hook
+    snap install --edge "test-snapd-with-default-configure${SUFFIX}"
```

Comment:
where is this snap? could you please add it to the repo? it should go in tests/lib/snaps/store/

Reply:
Thanks, added all base variants of the snap as suggested.